managed-identity-mysql/README.md

```diff
@@ -0,0 +1,57 @@
+# Access MySQL with managed identity

```

Comment:
`MySQL` -> `Azure Database for MySQL`

Reply:
Generic MySQL doesn't support managed identity. Let's be accurate. Please update all occurrences in this doc.